azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/AbstractWebAppMojo.java

```diff
@@ -348,6 +363,21 @@ public WebApp getWebApp() throws AzureAuthFailureException {
         return null;
     }
 
+    public DeploymentSlot getDeploymentSlot(final WebApp app, final String slotName) {
+        DeploymentSlot slot = null;
+        if (!StringUtils.isEmpty(slotName)) {

```

Comment:
Does StringUtils has `isNotEmpty()` method?

Reply:
Actually there is ^_^